src/Core/src/Graphics/IShape.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.Maui.Graphics
+{
+	/// <summary>
+	/// Provides a base definition class for shape elements, such as
+	/// Ellipse, Polygon, or Rectangle.
+	/// </summary>
+	public interface IShape
+	{
+		
```

Comment:
This should have `PathF CreatePath(RectangleF rect)` on it.

Reply:
Right, updated the interface definition.